pkg/apis/admissionregistration/types.go

```diff
@@ -486,14 +508,72 @@ type ValidatingAdmissionPolicyBindingSpec struct {
 	ValidationActions []ValidationAction
 }
 
-// ParamRef references a parameter resource
+// ParamRef describes how to locate the params to be used as input to
+// expressions of rules applied by a policy binding.
+//
+// Parameters may be identified by name and selector.
+//
+// If multiple params are found, they are all evaluated with the policy expressions
+// and the results are ANDed together.
+//
+// If name and `selector` are both specified, then admission will be evaluated
+// on params with the provided `name` OR `selector`. If multiple matches are found
```

Comment:
Stale information? Looks like this disagrees with validation rules and doc strings of name and selector which say that name and selector are mutually exclusive?

Reply:
this was stale, yeah. updated

Reply:
It should have been addressed with https://github.com/kubernetes/kubernetes/pull/119215#discussion_r1266411456.